environments/development/common/secrets.tf

```diff
@@ -293,3 +293,11 @@ module "fpo_search_sentry_dsn" {
   recovery_window = 7
   secret_string   = var.fpo_search_sentry_dsn
 }
+
+module "slack_notify_lambda_slack_webhook_url" {
```

Comment:
This enables us to get access to the secret but its not actually used by the module. Probably just remove it